### PR TITLE
MM32771 - improve tips and next steps cards

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR modifies the Next Tips & Steps section cards for the cloud users. Also adds an easiest way to get back to the MM chat workspace when the user is in the system console.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32771

#### Screenshots
Captura de pantalla 2021-03-12 a las 0 13 44

Captura de pantalla 2021-03-12 a las 0 16 04

Captura de pantalla 2021-03-12 a las 0 14 58

### CODE REVIEW

**Comment by annelieseklein** [comment]
Looks good! The arrow and "Back to <team>" should be in this blue. Also I've accounted for the hover state where "Back to <team>" is underlined

https://www.figma.com/file/1fUiqEEWuWO3Amnkdaodv5/Back-control-System-Console-MM32771?node-id=2%3A3

File: components/next_steps_view/next_steps_tips.tsx [hunk 1]
```diff
@@ -84,12 +87,22 @@ const getDownloadButtonString = () => {
...
 };
 
+const openAdminConsole = (isAdmin: boolean) => {
+    trackEvent(getAnalyticsCategory(isAdmin), 'click_admin_console');
+    browserHistory.push('/admin_console/');
+};
+
+const openIncidentsPlugin = (isAdmin: boolean, team: Team) => {
+    trackEvent(getAnalyticsCategory(isAdmin), 'click_open_incidents');
+    browserHistory.push(`/${team.name}/com.mattermost.plugin-incident-management/playbooks/`);
```
Comment (marianunez) [inline components/next_steps_view/next_steps_tips.tsx:103]: @lieut-data here we are linking to the playbooks backstage of IC. Just wanted to get your perspective if there was a better way to link there. Worried about plugin id changes and this link failing.

File: components/next_steps_view/index.ts [hunk 2]
```diff
@@ -22,12 +26,16 @@ function makeMapStateToProps() {
     const getCategory = makeGetCategory();
 
     return (state: GlobalState) => {
+        const teamId = LocalStorageStore.getPreviousTeamId(getCurrentUserId(state));
```
Comment (marianunez) [inline components/next_steps_view/index.ts:29]: Why `getPreviousTeamId`? Shouldn't be current team here?

File: components/next_steps_view/index.ts [hunk 3]
```diff
@@ -22,12 +26,16 @@ function makeMapStateToProps() {
...
+        const team = getTeam(state, teamId || '');
         return {
             currentUser: getCurrentUser(state),
             isAdmin: isCurrentUserSystemAdmin(state),
             preferences: getCategory(state, Preferences.RECOMMENDED_NEXT_STEPS),
             steps: getSteps(state),
             isFirstAdmin: isFirstAdmin(state),
+            isCloud: getLicense(state).Cloud === 'true',
```
Comment (marianunez) [inline components/next_steps_view/index.ts:37]: All onboarding section is only shown for cloud so not sure this check is needed here

File: components/next_steps_view/next_steps_tips.tsx [hunk 4]
```diff
@@ -181,50 +194,76 @@ export default function NextStepsTips(props: Props) {
...
                 </Card>
-                <Card expanded={true}>
-                    <div className='Card__body'>
-                        <div className='Card__image'>
-                            <img
-                                src={tipsNextStepsImage2}
-                            />
-                        </div>
-                        <h3>
+                { props.isCloud ?
```
Comment (marianunez) [inline components/next_steps_view/next_steps_tips.tsx:197]: These new steps will replace all current steps for Admins so no need to check for cloud

**Review by marianunez — CHANGES_REQUESTED** [review]

File: components/next_steps_view/next_steps_tips.tsx (hunk 1 above)
Comment (lieut-data) [reply components/next_steps_view/next_steps_tips.tsx:103]: I think this is the best we can do right now -- thanks for the heads up!

**Review by nickmisasi — COMMENTED** [review]
Nothing to add over Maria's comments. Just commenting so you can re-request review when you're done addressing them!

**Review by AGMETEOR — APPROVED** [review]

**Review by nickmisasi — APPROVED** [review]
LGTM!

**Review by annelieseklein — APPROVED** [review]

**Comment by marianunez** [comment]
@pablovelezvidal @ethervoid I believe this was not included in the cloud branch. Could you confirm ? If so, this needs to be cherry picked to be included in the March 24th release.

**Pull request merged**